url-shortener-microservice/server.js

```diff
@@ -25,13 +23,16 @@ app.get('/', (req, res) => {
   res.sendFile(process.cwd() + '/views/index.html');
 });
 
-app.post('/api/shorturl/new', urlHandler.addUrl);
-  
-app.get('/api/shorturl/:shurl', urlHandler.processShortUrl);
+// support legacy paths
+app.post(['/api/shorturl', '/api/shorturl/new'], urlHandler.addUrl);
 
+app.get(
+  ['/api/shorturl/:shurl', '/api/shorturl/:shurl'],
```

Comment:
What's the difference between these two?

Reply:
Oops, that's a mistake. Thanks for catching that.

Originally I was going to shorten this and the POST paths to `/api/...`, but decided against it in the end. With the `shorturl` resource, the GET path won't change, and just the POST path should support the legacy and new paths.

I'll go in and fix this now.